dandi/tests/fixtures.py

```diff
@@ -221,8 +233,56 @@ def fixture() -> Iterator[str]:
     return fixture
 
 
+def get_filtered_gitrepo_fixture(
+    url: str,
+    whitelist: List[str],
+) -> Callable[[], Iterator[str]]:
+    @pytest.fixture(scope="session")
+    def fixture() -> Iterator[str]:
+        skipif.no_network()
+        skipif.no_git()
+        path = tempfile.mktemp()  # not using pytest's tmpdir fixture to not
+        # collide in different scopes etc. But we
+        # would need to remove it ourselves
+        try:
+            lgr.debug("Cloning %r into %r", url, path)
+            runout = run(
```

Comment:
Add `check=True` to the `run()` call and skip the check of `runout.returncode`.  I also don't see a reason to capture the output.

Reply:
@TheChymera You missed this comment.

Reply:
sry, missed that. Should be solved now.